src/modules/packagemanagerconfiguration/src/lib/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+cmake_minimum_required(VERSION 3.2)
+
+set (CMAKE_CXX_STANDARD 17)
+
```

Comment:
C++ 17?

Use instead what the rest of OSConfig does:

cmake_minimum_required(VERSION 3.2)
set (CMAKE_CXX_STANDARD 11)

here and elsewhere

Reply:
The reason why we need C++ 17 is a [filesystem library](https://en.cppreference.com/w/cpp/filesystem) for performing operations on file systems.
C++ 17 was released already 5 years ago, so what are your concerns with using that version?

Reply:
C11 is the standard for OSConfig. C99 is the standard for the PnP SDK. Please change to C11.

Reply:
fixed as suggested